package.json

```diff
@@ -9,6 +9,7 @@
   },
   "dependencies": {
     "intl-tel-input": "^16.0.2",
-    "jquery": "^3.4.1"
+    "jquery": "^3.4.1",
+    "serve": "^13.0.2"
```

Comment:
Serve is not a production dependency, it should be placed in the `devDependencies`